### DISCUSSION TITLE
Changing number of columns in members

### DISCUSSION THREAD

**Discussion by hyeokhyen** [greenelab/lab-website-template#276] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/KwonVitalLab/KwonVitalLab.github.io

### Version of Lab Website Template you are using

1.1.0

### Description

Currently, the default number of columns on members layout is 4. How can we change this to 5 or other number of columns?
I will appreciate your help!

**Comment by vincerubinetti** [comment, marked as answer]
The members page isn't really columns, it's just regular text wrapping, like in a word document. You can observe this on smaller screen sizes, depending on the number of members:

<img src="https://github.com/user-attachments/assets/3a8e87cd-b6c9-4d6c-b846-9778e1ba72da" height="200"/>

Notice that there aren't strict "columns", because the third member is in the center of the other two. The member circles have a set size, and we simply try to fit as many on one line as we can before wrapping (and center-aligning) them. It is important to understand this. Not everyone will have the same screen size as you, and the layout of your site has to change to accommodate that. 

Having said that, you can get what you're after in a few different ways:

- Increase the size of the `$page` variable in `/_styles/section.scss` to increase the available width we can fill up before wrapping. This will affect all sections on every page though.
- On the specific [section break](https://greene-lab.gitbook.io/lab-website-template-docs/basics/components/section) on your team page, set the `size` to `wide`, which will make just that section take up the full width of the screen (which, again, will be different for different visitors of your site).
- Decrease the `width`s under `.portrait {` and `.portrait[data-style="small"] {` in `/_styles/portrait.scss`. The circles will become smaller, and thus more will be able to fit on one line.

**Reply by hyeokhyen** [reply]
Ah great. Thank you. Setting section size to wide solved it :) Thank you for your quick reply!
